js/ui/calendar/ui.calendar.js

```diff
@@ -1180,8 +1218,8 @@ const Calendar = Editor.inherit({
     },
 
     _optionChanged: function(args) {
-        let value = args.value;
-        let previousValue = args.previousValue;
+        const value = args.value;
+        const previousValue = args.previousValue;
```

Comment:
May we use destructuring assignment here?
const { value, previousValue } = args;

Reply:
changed this

just to note: it's actually was not my change, I just removed value/previousValue changing below and lint replaced let with const on ctrl+s

Reply:
Yes, but you change let to const and use them in you code. The destructuring assignment `const { value, previousValue, name } = args;` looks appropriate here. It's recomendation only and you should to take a final decision to do or not to do.